logging/tests/unit/handlers/transports/test_background_thread.py

```diff
@@ -282,6 +284,22 @@ def test__thread_main(self):
         self.assertEqual(worker._cloud_logger._batch.commit_count, 2)
         self.assertEqual(worker._queue.qsize(), 0)
 
+    def test__thread_main_no_python_logger(self):
+        from google.cloud.logging.handlers.transports import background_thread
+
+        worker = self._make_one(_Logger(self.NAME), include_logger_name=False)
```

Comment:
Add an assertion, e.g.:

```suggestion
        worker = self._make_one(_Logger(self.NAME), include_logger_name=False)
        self.assertFalse(worker._include_logger_name)
```

Reply:
Done!